README.md

```diff
@@ -107,7 +107,7 @@ To run the Docker version, follow these steps:
 and open the directory in the console. Then run `git clone https://github.com/Human-Connection/Clock-of-Change-API.git` to clone the repository to this directory.
 2. Go to the newly created Clock-of-Change-API directory (`cd Clock-of-Change-API` in the console)
 3. Run `docker-compose up`. This will build the Docker container on first startup and run it. This can take a while, but after some time you should see the Clock of Change ticking.
-4. To create an initial API key and sample entries, run `docker-compose exec db seed.sh`. The initial API key you can use with your requests will have the value `secret`.
+4. To create an initial API key and sample entries, run `docker-compose exec db bash seed.sh`. The initial API key you can use with your requests will have the value `secret`.
```

Comment:
This looks wrong. You sync `seed.sh` in a folder under `$PATH` making `seed.sh` an executable program.

Reply:
Can change it back, if that is better.
Unfortunately I get the error: 
```
OCI runtime exec failed: exec failed: container_linux.go:344: starting container process caused "no such file or directo
ry": unknown
```
when trying to run it from $PATH directly. With running first bash and then passing seed.sh as an argument, the seed.sh can successfully be found in the $PATH.

This could be a problem with my local environment, as I use the Windows Git Bash as my console.